amclient/amclientargs.py

```diff
@@ -37,8 +36,16 @@
     help="Relative path to a file in an Archivematica package",
     type=None,
 )
-
-# Reusable option constants (for CLI).
+SPACE_UUID = Arg(
+    name="space_uuid", help="UUID of the space to add the location to", type=None
+)
+SPACE_RELATIVE_PATH = Arg(
+    name="space_relative_path", help="Path relative to a storage space", type=None
+)
+LOCATION_PURPOSE = Arg(
+    name="location_purpose", help="Purpose of storage space location", type=None
```

Comment:
Do you think it's worth to list the ids of the valid location purposes here?

Reply:
Hmm, I could do, I return it if the user gets it wrong! :stuck_out_tongue: but let's try adding it here too as it might help too.